components/eamxx/src/diagnostics/tests/field_at_height_tests.cpp

```diff
@@ -128,22 +148,22 @@ TEST_CASE("field_at_height")
     print(" -> Testing with z_tgt coinciding with a z level\n");
```

Comment:
Can you add a test like
```suggestion
REQUIRE_THROWS (run_diag(s_mid,z_mid,"foobar"));
```
to verify the check on the surf_ref type works correctly?